packages/web/package.json

```diff
@@ -32,6 +32,7 @@
     "react-ens-name": "^0.2.3",
     "react-modal": "^3.14.3",
     "react-sketch-canvas": "^5.3.4",
+    "react-use": "^17.3.1",
```

Comment:
This library has a ton of great hooks and is generally what I reach for for a lot of lower level stuff. Could probably swap out other deps for similar hooks that exist in this one.